pkg/alerter/alerter.go

```diff
@@ -0,0 +1,71 @@
+package alerter
+
+import (
+	"fmt"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/sirupsen/logrus"
+)
+
+type Alerter struct {
+	Alerts   Alerts
+	AlertsCh chan Alerts
+}
+
+func NewAlerter() *Alerter {
+	alerts := make(Alerts)
+	alertsCh := make(chan Alerts)
+
+	go func() {
```

Comment:
for readability and isolation I would prefer to see the goroutine as a Alerter method i think

Reply:
Done 442d3ed